### DISCUSSION TITLE
Change attachment limit size?

### DISCUSSION THREAD

**Discussion by WavedirectTel** [docmost/docmost#552] [Help / Questions] [answered]
Some larger PDF's may need to be attached.  Would be nice to be able to have larger, also some form of file browser/uploader? Then we can attach docs from a repo

"File exceeds the 50.00 MB attachment limit"

**Comment by Philipinho** [comment, marked as answer]
You can increase the file upload limit with this env variable.
`FILE_UPLOAD_SIZE_LIMIT=100mb`

**Reply by HsynNH** [reply]
Hi, I am trying to set the variable inside the .yml file through SSH. I inserted FILE_UPLOAD_SIZE_LIMIT=100mb under service:environment but when restarting it says: "yaml: line 14: could not find expected ':'". Could you help me with it?

**Reply by Philipinho** [reply]
Try `FILE_UPLOAD_SIZE_LIMIT: 100mb`

**Comment by WavedirectTel** [comment]
Thank you, I was looking in the docs to see if there was a variable.  Much appreciated!
